01-Login/src/components/PrivateRoute.js

```diff
@@ -26,9 +27,6 @@ const PrivateRoute = ({ component: Component, path, location, ...rest }) => {
 PrivateRoute.propTypes = {
   component: PropTypes.oneOfType([PropTypes.element, PropTypes.func])
     .isRequired,
-  location: PropTypes.shape({
-    pathname: PropTypes.string.isRequired
-  }).isRequired,
   path: PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.arrayOf(PropTypes.string)
```

Comment:
Going the other way, I noticed that `PropTypes` is not included in the code snippet on the docs side - do you think that's worth syncing up as well?

Reply:
Same Q for sample 02 below 👇

Reply:
Yeah, happy to add it as long as you don't think it would over complicate the QS?

Reply:
Thinking about it, you're right. It adds bulk to the QS snippet without any real benefit to the auth part. Happy to leave it out.